pkgs/by-name/al/alp/package.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, buildGoModule
+, fetchFromGitHub
+, runCommand
+, alp
+}:
+buildGoModule rec {
+  pname = "alp";
+  version = "1.1.17";
+  src = fetchFromGitHub {
+    owner = "gernotfeichter";
+    repo = pname;
+    rev = version;
+    hash = "sha256-7lyWu1bVn7UwLb/Em6VBbg3FrMyxGjebxt5gJhm/xpI=";
+  };
+  vendorHash = "sha256-a2CQZKN/rPWh/Pn9gXfSArTCcGST472tsz1Kqm7M4vM=";
+  sourceRoot = "${src.name}/linux";
+  checkPhase = ''
+    go test -run Test_main_init
+  '';
+  buildPhase = ''
+    go build -o $GOPATH/bin/${pname} main.go
+  '';
+  passthru.tests = {
+    simple = runCommand
+      "${pname}-test"
+      {}
+      ''
+      ${alp}/bin/${pname} version > $out
+      cat $out | grep '${version}'
+      '';
+  };
+  meta = with lib; {
+    description = "alp - android-linux-pam";
+    homepage = "https://github.com/gernotfeichter/alp";
```

Comment:
```suggestion
    homepage = "A convenient authentication method that lets you use your android device as a key for your Linux machine";
```

Reply:
I see the intention must have been to update the description field and not the homepage, so I commited that change instead. And squashed all the remaining edits together!